server/graphql/schema.ts

```diff
@@ -53,6 +58,23 @@ const typeDefs = gql`
         requestGroup(id: ID): RequestGroup
         requestGroups: [RequestGroup]
     }
+    type Mutation {
+        createRequest(request: RequestInput): ServerResponse
+        updateRequest(request: RequestInput, id: ID): ServerResponse
+        softDeleteRequest(id: ID): ServerResponse
+    }
+    input ClientInput {
+        _id: ID
+        clientId: String
+        firstName: String
+        lastName: String    
+    }
+    input RequestInput {
+        client: ClientInput
```

Comment:
You should be able to create a Request without creating a new Client. Please change the type of client to ID.

Reply:
Done